network/horizon/horizon-providers.mdx

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 70
+title: Ecosystem Horizon Providers
+---
+
+## Ecosystem Horizon Providers
+
+Multiple infrastructure providers have made Horizon services available, and offer plans ranging from free to high throughput endpoints. These providers can be used for development, testing, and production.
+
+These providers allow access to the Testnet and Mainnet networks.
+
+| Provider                                                                           | Access                                              | Futurenet     | Testnet   | Mainnet     |
+| ---------------------------------------------------------------------------------- | --------------------------------------------------- | ------------- | --------- | ----------- |
+| [Blockdaemon\*](https://docs.blockdaemon.com/reference/supported-stellar-methods) | [Access](https://www.blockdaemon.com/apply/soroban) | Not available | Available | Available   |
+| [QuickNode\*](https://www.quicknode.com/docs/stellar)                             | [Access](https://www.quicknode.com/chains/stellar)  | Not available | Available | Available   |
+| [Ankr](https://www.ankr.com/rpc/stellar/) | [Access](https://www.ankr.com/rpc/stellar/) | Not available | Available | Available   |
+
+
+\*_Please note that both Blockdaemon and Quicknode combine Horizon and RPC in the same endpoint._
+
+## SDF Provided Horizon (Futurenet and Testnet) {#sdf-provided-horizon}
+
+SDF will not be providing a publicly available Horizon endpoints that are rate-limited and subjected to changes including truncation of history. Developers are encouraged to select an ecosystem provider. In some cases, projects may choose to run their own Horizon instance.
```

Comment:
This sentence seems to contradict itself. It says "SDF will not be providing a publicly available Horizon endpoint" while also saying "the endpoint is rate-limited and subject to changes including truncation of history". Do we mean to say "SDF provides a Horizon endpoint but is rate limited and subject to changes such as truncation of history"?

Reply:
Sounds good updating